.env

```diff
@@ -1 +1,7 @@
 SERVER_PORT=4550
+AZURE_TENANT_ID=
```

Comment:
typically .env is ignored actually. Other way round to what you have here. check out pre-portal